scrapers_next/or/committee.py

```diff
@@ -0,0 +1,61 @@
+from spatula import XPath, URL, HtmlListPage, HtmlPage, CSS
+from openstates.models import ScrapeCommittee
+import re
+
```

Comment:
PEP 8 pythonic styling encourages two lines of space between a class and anything else. Would be good to add a space here and also more down near the bottom of the file to have two spaces between those other classes.